CONTRIBUTING.md

```diff
@@ -267,3 +267,21 @@ We encourage you to support others in their Serverpod projects by sharing your k
 ## Contributing with issues
 
 Help us make Serverpod better by filing [issue](https://github.com/serverpod/serverpod/issues/new/choose) for bugs, feature requests, or improvements. When filing an issue, please provide as much information as possible to help us understand the problem or suggestion.
+
+## Coding style
+
+### Naming conventions
```

Comment:
I think from your comment on https://github.com/serverpod/serverpod_docs/pull/268, we might also want to add a small dictionary.

Like the open question what is an endpoint? Class or method, or both? In gRPC they use service (for the container) and method.